pkg/server/thick_cni_test.go

```diff
@@ -46,17 +46,17 @@ const suiteName = "Thick CNI architecture"
 type fakeExec struct{}
 
 // ExecPlugin executes the plugin
-func (fe *fakeExec) ExecPlugin(ctx context.Context, pluginPath string, stdinData []byte, environ []string) ([]byte, error) {
+func (fe *fakeExec) ExecPlugin(_ context.Context, _ string, _ []byte, _ []string) ([]byte, error) {
 	return []byte("{}"), nil
 }
 
 // FindInPath finds in path
-func (fe *fakeExec) FindInPath(plugin string, paths []string) (string, error) {
+func (fe *fakeExec) FindInPath(_ string, _ []string) (string, error) {
```

Comment:
Make peace with the linter ? It is complaining these attributes are not being used.

Thus we tell the compiler we don't care enough to name them. 

... this needs to happen because this is a method of an interface.

Reply:
Thanks Miguel, also talked to Tomo about the changes. I think we're good to merge here